public/application.js

```diff
@@ -77,9 +78,18 @@ function initialize() {
       }
     },
     loadUpcomingMeetup: function () {
-      const upcomingMeetup = this.meetups.find(
-        (meetup) => new Date(meetup.datetime) > new Date()
-      )
+      const now = moment()
+      const endOfDay = moment().endOf('day')
+
+      const upcomingMeetup = this.meetups.find((meetup) => {
+        const meetupDate = moment(meetup.datetime)
+
+        return (
+          now.isSameOrBefore(endOfDay, 'day') &&
```

Comment:
I'm not sure about this line ``now.isSameOrBefore(endOfDay, 'day')``. When this return false ?

```suggestion
```

Reply:
This condition is false when the current time is out of range for the day, that is, after the end of the day.

Reply:
After speaking with @dawidrudnik we fixed this